test/integration/docker_test.go

```diff
@@ -123,12 +123,11 @@ func validateContainerdSystemd(ctx context.Context, t *testing.T, profile string
 
 // validateCrioSystemd makes sure the --force-systemd flag worked with the cri-o container runtime
 func validateCrioSystemd(ctx context.Context, t *testing.T, profile string) {
-	rr, err := Run(t, exec.CommandContext(ctx, Target(), "-p", profile, "ssh", "cat /etc/crio/crio.conf"))
+	rr, err := Run(t, exec.CommandContext(ctx, Target(), "-p", profile, "ssh", "cat /etc/crio/crio.conf.d/02-crio.conf"))
 	if err != nil {
 		t.Errorf("failed to get cri-o cgroup driver. args %q: %v", rr.Command(), err)
 	}
-	// cri-o defaults to `systemd` if `cgroup_manager` not set, so we remove `cgroup_manager` on force
-	if strings.Contains(rr.Output(), "cgroup_manager = ") {
+	if strings.Contains(rr.Output(), "cgroup_manager = \"systemd\"") {
```

Comment:
```suggestion
	if !strings.Contains(rr.Output(), "cgroup_manager = \"systemd\"") {
```

Also just want to negate this as we only want to throw an error if the config doesn't contain `systemd` as it's the `TestForceSystemdFlag` test

Reply:
sorry, I misread your earlier comment

Reply:
No worries, if the tests pass this looks good to me, thanks for all the fixes